src/libraries/System.Globalization/tests/CompareInfo/CompareInfoTests.cs

```diff
@@ -357,6 +357,7 @@ public static void LcidTest(string cultureName, int lcid)
         [MemberData(nameof(SortKey_Kana_TestData))]
         public void SortKeyKanaTest(CompareInfo compareInfo, string string1, string string2, CompareOptions options, int expected)
         {
+            [ActiveIssue("https://github.com/dotnet/runtime/issues/36672", TestPlatforms.Android)]
```

Comment:
Not a test.  Move it up to `SortKeyKanaTest`

Reply:
Fixed my script to put active issues above the test function definition, ignoring any calls of the test function